src/object_store/src/object/error.rs

```diff
@@ -27,8 +27,13 @@ use tokio::sync::oneshot::error::RecvError;
 #[derive(Error, Debug, thiserror_ext::Box, thiserror_ext::Construct)]
 #[thiserror_ext(newtype(name = ObjectError, backtrace, report_debug))]
 pub enum ObjectErrorInner {
-    #[error("s3 error: {0}")]
-    S3(#[source] BoxedError),
+    #[error("s3 error: {inner}")]
+    S3 {
+        // TODO: remove this after switch s3 backend to opendal
```

Comment:
0.0 Why? Any infomation?

Reply:
we can move to opendal::Error after switching to opendal